build/installers/installers-tests/pom.xml

```diff
@@ -0,0 +1,36 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+        <groupId>com.liferay.ide.build</groupId>
+        <artifactId>installers</artifactId>
+        <version>3.2.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>installers.tests</artifactId>
+
+    <name>Liferay IDE Installers Tests</name>
```

Comment:
These tests should be added to the installer profile.  We don't want these tests run if the installer profile has executed.